src/wallet/spend.cpp

```diff
@@ -895,13 +884,7 @@ static util::Result<CreatedTransactionResult> CreateTransactionInternal(
     CAmount selection_target = recipients_sum + not_input_fees;
 
     // Get available coins
-    auto available_coins = AvailableCoins(wallet,
-                                              &coin_control,
-                                              coin_selection_params.m_effective_feerate,
-                                              1,            /*nMinimumAmount*/
-                                              MAX_MONEY,    /*nMaximumAmount*/
-                                              MAX_MONEY,    /*nMinimumSumAmount*/
-                                              0);           /*nMaximumCount*/
+    auto available_coins = AvailableCoins(wallet, &coin_control, coin_selection_params.m_effective_feerate);
```

Comment:
For reviewers: These hardcoded values can be removed as they are now the defaults of `AvailableCoinsParams`.